api/src/opentrons/protocol_api_experimental/instrument_context.py

```diff
@@ -63,7 +64,31 @@ def dispense(self,  # noqa: D102
                  volume: Optional[float] = None,
                  location: Union[types.Location, Well] = None,
                  rate: float = 1.0) -> InstrumentContext:
-        raise NotImplementedError()
+        """Dispense a volume of liquid (in microliters)."""
+        if rate != 1:
+            raise NotImplementedError("Flow rate adjustment not yet supported in PE.")
+
+        if volume is None:
+            raise NotImplementedError("Volume tracking not yet supported in PE.")
```

Comment:
```suggestion
        if not volume:
            raise NotImplementedError("Volume tracking not yet supported in PE.")
```

Unfortunately, current APIv2 behavior is to dispense the current volume if `volume` is `None` *or 0.* For now, we should preserve that behavior here.

https://github.com/Opentrons/opentrons/blob/df68ea2da42b99334eee17077ed9dc3d3f3bf7f8/api/src/opentrons/protocol_api/instrument_context.py#L291

#6741 documented this for other `InstrumentContext` methods, but it looks like I missed `dispense()`. Sorry!

Reply:
I see what you're saying, but I kind of prefer the explicitness of `volume is None`. Perhaps we could do `volume is None or volume == 0`?

On the JS-side, we have linters that yell at you if you try to infer a falsey value from a number that may be `null`, and I think the same rule should apply here with `None`

Reply:
Oh! I did not realize it does that for volume == 0. Thanks @SyntaxColoring.
I like @mcous's suggestion of `volume is None or volume == 0`. I think `not volume` makes it easy to overlook the 0 volume case as I just did.